modules/swagger-codegen/src/main/resources/typescript-angular/api.mustache

```diff
@@ -39,7 +39,7 @@ namespace {{package}} {
          * {{notes}}
          {{#allParams}}* @param {{paramName}} {{description}}
```

Comment:
please revert the changes on this file, since the changes now only apply to angular2-typescript